File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ListItem.java
package com.pkmnapps.activitydo.dataclasses;

public class ListItem {
    String uid, content, lid;
    Boolean checked;

    public ListItem(String uid, String lid, String content, Boolean checked) {
        this.uid = uid;
        this.lid = lid;
        this.content = content;
        this.checked = checked;
    }

    public String getLid() {
        return lid;
    }

    public void setLid(String lid) {
        this.lid = lid;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Boolean getChecked() {
        return checked;
    }

    public void setChecked(Boolean checked) {
        this.checked = checked;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ListItem.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class ListItem(var uid: String?, var lid: String?, var content: String?, var checked: Boolean?) {
    fun getLid(): String? {
        return lid
    }

    fun setLid(lid: String?) {
        this.lid = lid
    }

    fun getUid(): String? {
        return uid
    }

    fun setUid(uid: String?) {
        this.uid = uid
    }

    fun getContent(): String? {
        return content
    }

    fun setContent(content: String?) {
        this.content = content
    }

    fun getChecked(): Boolean? {
        return checked
    }

    fun setChecked(checked: Boolean?) {
        this.checked = checked
    }
}